test/expected/noisy.out

```diff
@@ -1,6 +1,7 @@
 LOAD 'pg_diffix';
 SET pg_diffix.noise_layer_sd = 7;
-SET pg_diffix.low_count_layer_sd = 3;
+SET pg_diffix.low_count_layer_sd = 2;
+SET pg_diffix.low_count_min_threshold = 2;
```

Comment:
Both changes effectively raised the threshold mean, so here we're lowering it back to have same tests.